### PR TITLE
Corrected script for some item #1

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: 

* **Description of Pull Request**:

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -144704,6 +145388,79 @@ Body:
...
+    Type: Armor
+    Weight: 200
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus bMdef,10;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:145403]:
```suggestion
      bonus bUnbreakableShoes;
      bonus bMdef,10;
```

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -144704,6 +145388,79 @@ Body:
...
+    Weight: 200
+    Slots: 1
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus bMdef,10;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:145440]:
```suggestion
      bonus bUnbreakableShoes;
      bonus bMdef,10;
```

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -145083,6 +145861,42 @@ Body:
...
+    Defense: 12
+    Slots: 1
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      .@val = getskilllv("GN_CARTBOOST");
```
Comment (Atemo) [inline db/re/item_db_equip.yml:145878]: If I'm not wrong `精錬 : 可 破損 : しない`
```suggestion
      .@val = getskilllv("GN_CARTBOOST");
      bonus bUnbreakableShoes;
```

File: db/re/item_db_equip.yml [hunk 4]
```diff
@@ -145503,6 +146317,31 @@ Body:
...
+    Weight: 1000
+    Defense: 40
+    Slots: 1
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
```
Comment (Atemo) [inline db/re/item_db_equip.yml:146333]:
```suggestion
      .@r = getrefine();
      bonus bUnbreakableShoes;
```

File: db/re/item_combos.yml [hunk 5]
```diff
@@ -23606,3 +23606,432 @@ Body:
...
+      bonus2 bMagicAddSize,Size_All,1*(.@r/2);
+      if (.@r>=12)
+         bonus2 bSkillCooldown,"SO_VARETYR_SPEAR",-1000;
+  - Combos:
+      - Combo:
+          - aegis_480166
+          - Old_Magic_Stone_Hat
+    Script: |
+      .@r = getequiprefinerycnt(EQI_GARMENT);
+      bonus2 bMagicAddSize,Size_All,1*(.@r/2);
```
Comment (Atemo) [inline db/re/item_combos.yml:24026]:
```suggestion
      bonus2 bMagicAddSize,Size_All,(.@r/2);
```

File: db/re/item_db_equip.yml [hunk 6]
```diff
@@ -145011,6 +145768,27 @@ Body:
...
+    Weight: 1000
+    Defense: 18
+    Slots: 1
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 99
+    Refineable: true
+    Script: |
+      .@r = getrefine();
```
Comment (Atemo) [inline db/re/item_db_equip.yml:145784]:
```suggestion
      .@r = getrefine();
      bonus bUnbreakableShoes;
```

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_db_equip.yml (hunk 6 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:145784]: Thanks!

File: db/re/item_combos.yml (hunk 5 above)
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 4 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:146333]: Thanks!

File: db/re/item_db_equip.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:145878]: OK, Thanks!

File: db/re/item_db_equip.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:145440]: Thanks!

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:145403]: Thanks!

File: db/re/item_combos.yml [hunk 7]
```diff
@@ -23606,3 +23606,432 @@ Body:
...
+      skill "AB_JUDEX",.@r,1;
+  - Combos:
+      - Combo:
+          - aegis_450181
+          - aegis_300348
+    Script: |
+      .@r = min(10,getequiprefinerycnt(EQI_ARMOR));
+      bonus2 bMagicAtkEle,Ele_Holy,15;
+      bonus2 bResEff,Eff_Freeze,10000;
+      skill "AB_JUDEX",.@r,1;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_combos.yml [hunk 8]
```diff
@@ -23606,3 +23606,432 @@ Body:
...
+      skill "AB_JUDEX",1;
+  - Combos:
+      - Combo:
+          - aegis_450181
+          - Grave_Amon_Ra_Card
+    Script: |
+      .@r = min(10,getequiprefinerycnt(EQI_ARMOR));
+      bonus2 bMagicAtkEle,Ele_Holy,50;
+      bonus2 bResEff,Eff_Freeze,10000;
+      skill "AB_JUDEX",.@r,1;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_combos.yml [hunk 9]
```diff
@@ -23606,3 +23606,432 @@ Body:
       bonus bSmatk,2;
       bonus2 bAddClass,Class_All,10;
       bonus bMatkRate,10;
+  - Combos:
+      - Combo:
+          - RingOfThought_TW
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: OK, Thanks!

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**